packages/components/addon/components/hds/pagination-bar/index.hbs

```diff
@@ -1,4 +1,73 @@
 {{! ADD YOUR TEMPLATE CONTENT HERE }}
 <div class={{this.classNames}} ...attributes>
-  {{yield}}
-</div>
\ No newline at end of file
+  <div class="hds-pagination-bar__total-count">1–10 of 100</div>
+
+  <nav class="hds-pagination-bar__pagination">
+    <ul class="hds-pagination-bar__page-list">
+      <li class="hds-pagination__page hds-pagination__page--previous">
+        <a class="hds-pagination__page-link" aria-label="Previous page" href="/page-?">
+          <FlightIcon @name="chevron-left" class="hds-pagination__nav-icon" /><span
+            class="hds-pagination__page-label"
+          >Previous</span>
+        </a>
+      </li>
+      <li class="hds-pagination__page hds-pagination__page--current">
+        <a class="hds-pagination__page-link" href="/page-1">1</a>
+      </li>
+      <li class="hds-pagination__page">
+        <a class="hds-pagination__page-link" href="/page-2">2</a>
+      </li>
+      <li class="hds-pagination__page">
+        <a class="hds-pagination__page-link" href="/page-3">3</a>
+      </li>
+      <li class="hds-pagination__page">
+        <a class="hds-pagination__page-link" href="/page-4">4</a>
+      </li>
+      <li class="hds-pagination__page">
+        <a class="hds-pagination__page-link" href="/page-5">5</a>
+      </li>
+      <li class="hds-pagination__page">
+        <span class="hds-pagination__page-link hds-pagination__page-link--ellipsis">...</span>
+      </li>
+      <li class="hds-pagination__page">
+        <a class="hds-pagination__page-link" href="/page-10">10</a>
+      </li>
+      <li class="hds-pagination__page hds-pagination__page--next">
+        <a class="hds-pagination__page-link" href="/page-?" aria-label="Next page">
+          <span class="hds-pagination__page-label">Next</span><FlightIcon
+            @name="chevron-right"
+            class="hds-pagination__nav-icon"
+          />
+        </a>
+      </li>
+    </ul>
+  </nav>
+
+  <div class="hds-pagination-bar__page-size">
+    <Hds::Form::Select::Field class="hds-pagination-bar__page-size-select" as |F|>
+      <F.Label>Items per page</F.Label>
+      <F.Options>
+        <option value="Kubernetes">10</option>
+        <option value="Other">30</option>
+        <option value="Other">50</option>
+      </F.Options>
+    </Hds::Form::Select::Field>
```

Comment:
Should I be using this component here? Right now I have rough hacky CSS styles for it to match Figma mocks which I plan to refactor...

Reply:
Seems like we should be able to use the component, if you scope the styles to this new parent component it should work out.

Reply:
I can see... I was also wondering if there would be a possibility to add a variation to this component for a "compact" version or something but maybe that wouldn't make sense since so far this is just a one-off use case.

Reply:
The select only has different (decreased) padding and smaller font size. In Figma, it is a separate component to avoid unexpected behaviour from the main component and ensure we the component doesn't have weird resets

Reply:
If I use `Hds::Form::Select::Base` then it seems I have to create my own html `label` tag for the "Items per page" text to label it and then I have to figure out how best to add an id to associate the label element with the `select` or else I could nest the `Hds::Form::Select::Base` inside the `label` element but don't know if we like doing that... 

Anyway not sure if it's the best solution yet to use that component over `Hds::Form::Select::Field` which lets you specify the label element as part of it and then automatically associates them.

Reply:
@KristinLBradley let's figure out this is a follow-up PR (or in the main one). for now let's not consider it a blocker (I personally think we should not mix the two, even if this means duplicating some code, but I am interested to hear also @alex-ju opinion on this)

Reply:
In theory, I'm incline to favor a variant for the select component ('compact' or so, which would mean adding a modifier class). Now, if we don't want to make this variant public (and I can see reasons), I'd fall back on using `Form::Select::Base`

Reply:
Thanks @alex-ju, I would personally favor adding a "Compact" variant for the select component but if the use case is too one off and we don't want it used widely elsewhere then that would be a reason not to.

Using `Hds::Form::Select::Base` vs. `Hds::Form::Select::Field` though means I have to recreate the label element and if we need to add an id to the select to reference it in the label's for attribute then that's a further complication...